Godot-mono not compiling

**Godot version:**
Latest master.

**OS/device including version:**
Windows 10 AMD 64 VS 2017


**Issue description:**
Cant build godot-mono from source:
```
scons: Reading SConscript files ...
Configuring for Windows: target=debug, bits=default
 Found MSVC version 14.1, arch amd64, bits=64
WebM SIMD optimizations are disabled. Check if your CPU architecture, CPU bits or platform are supported!
Checking for C header file mntent.h... (cached) no
scons: done reading SConscript files.
scons: Building targets ...
[  5%] [91mLinking Static Library [95m==> [93mmain\main.windows.tools.lib[0m
[ 93%] progress_finish(["progress_finish"], [])
[ 96%] Linking Program        ==> bin\godot.windows.tools.mono.exe
LINK : fatal error LNK1181: could not open input file "mono-2.0-sgen.windows.tools.lib"
scons: *** [bin\godot.windows.tools.mono.exe] Error 1181
scons: building terminated because of errors.
```
MONO64_PREFIX is correct.
**Steps to reproduce:**
Compile.

**Minimal reproduction project:**
Latest master, no changes

--- a/modules/mono/config.py
+++ b/modules/mono/config.py
@@ -85,7 +85,7 @@ def configure(env):
             if not os.path.isfile(os.path.join(mono_lib_path, mono_static_lib_name + lib_suffix)):
                 raise RuntimeError('Could not find static mono library in: ' + mono_lib_path)
 
-            if os.getenv('VCINSTALLDIR'):
+            if env.msvc:
                 env.Append(LINKFLAGS=mono_static_lib_name + lib_suffix)
 
                 env.Append(LINKFLAGS='Mincore' + lib_suffix)
@@ -100,7 +100,7 @@ def configure(env):
             if not mono_lib_name:
                 raise RuntimeError('Could not find mono library in: ' + mono_lib_path)
 
-            if os.getenv('VCINSTALLDIR'):
+            if env.msvc:
                 env.Append(LINKFLAGS=mono_lib_name + Environment()['LIBSUFFIX'])
             else:
                 env.Append(LIBS=mono_lib_name)

